keyboards/soda/mango/config.h

```diff
@@ -0,0 +1,90 @@
+/*
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+// enable NKRO_ENABLE
+#define FORCE_NKRO
+// The pin connected to the data pin of the LEDs
+#define RGB_DI_PIN A2
```

Comment:
This is changed in develop, and lands on master in a week.  To ensure that it compiles: 
```suggestion
#define RGB_DI_PIN A2
#define WS2812_DI_PIN A2
```